ls1intum/Artemis
# `Development`: Improve setup instructions

### Checklist
#### General

### Motivation and Context
I set up a local instance of Artemis and had some minor problems:
- There were slightly outdated instructions (e.g. changed configuration page names in Bitbucket).
- I had some minor undocumented issues which I had to figure out myself.

Although none of these issues made it impossible to set up Artemis locally, I wanted to provide some additional / changed instructions for added convenience of future developers.

### Description
I created notes about every problem I had while setting up a local instance of Artemis and changed the Setup documentation accordingly afterwards. While doing that, I also noticed a minor error in the Sphinx build instructions and fixed that as well. 

I updated a screenshot and changed the filename to be more consistent with the filenames of other screenshots in the same folder.

I did not modify any actual code.

https://artemis-platform--6091.org.readthedocs.build/en/6091/dev/setup/

### Steps for Testing

Prerequisites:
- Sphinx installation

1. Build the documentation (see `docs/README.md`)
2. Open the documentation page and navigate to the "Setup Guide".
3. Read through the changes.

### Review Progress






#### Code Review
